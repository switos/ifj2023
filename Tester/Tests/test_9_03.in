let x : Int = 3
if (x > 5) {
    let y = 10
} else {}
let result = y + 5
// 'y' is used outside of its scope
