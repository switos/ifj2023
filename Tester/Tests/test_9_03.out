Expected Output:

Expected Return Code: 9
